FlowCal/excel_ui.py

```diff
@@ -181,16 +188,16 @@ def process_beads_table(beads_table,
     mef_transform_fxns : OrderedDict
         A dictionary of MEF transformation functions, indexed by
         ``beads_table.index``.
-    mef_outputs : OrderedDict
-        A dictionary with the intermediate results of the generation of the
-        MEF transformation functions, indexed by ``beads_table.index``.
-        Only included if `full_output` is True.
+    mef_outputs : list
+        A list with intermediate results of the generation of the MEF
+        transformation functions, indexed by ``beads_table.index``. Only
+        included if `full_output` is True.
```

Comment:
I changed it to make it more consistent with `beads_samples`. The few times that I've had to use `mef_outputs` (like for some analysis for the paper), I've had to iterate through it along with `beads_samples` and `beads_table`. This is annoying if `mef_outputs` is a dictionary. However, one could do `mef_outputs.items()` and, because it's an `OrderedDict`, the items would be in order. But I haven't had a single case in which I needed the dictionary functionality. Another reason is simplicity. I think it gets confusing if `beads_table` is a dataframe, `beads_samples` is a list, and `mef_outputs` is an ordered dictionary.